finetune/utils.py

```diff
@@ -13,6 +14,18 @@
 from finetune import config
 
 
+def datatype(typename, field_names):
```

Comment:
need to credit stackoverflow here

Reply:
This will ignore misspelt or extra key names. Also, I dont understand, named tuples accept kwargs??

Reply:
Im sure ive just missed something here, but I dont know what advantage this gives over named tuple.